R/barplot.R

```diff
@@ -20,10 +19,10 @@ melt_metacoder <- function(obj) {
 #' @title Transform Metacoder Dataframe
 #' @description Transform the dataframe abundance values to percent 100.
 #' @param melted_df A "melted" dataframe from the metacoder object's data.
-#' @param tax_level The taxonomic level.
 #' @return Returns a transformed dataframe.
 #' @importFrom dplyr filter group_by summarize mutate enquo quo_name
 #' @importFrom stats na.omit
+#' @rdname stacked_barplot
 transform_metacoder <- function(melted_df, tax_level) {
```

Comment:
```suggestion
melted_proportions <- function(melted_df, tax_level) {
```

Reply:
quick question here...

Would you like me to change this function's name or the parameter to `melted_proportions`

```r
#' @title Transform Metacoder Dataframe
#' @description Transform the dataframe abundance values to percent 100.
#' @param melted_df A "melted" dataframe from the metacoder object's data.
#' @return Returns a transformed dataframe.
#' @importFrom dplyr filter group_by summarize mutate enquo quo_name
#' @importFrom stats na.omit
#' @family Data Manipulators
#' @rdname stacked_barplot
transform_metacoder <- function(melted_df, tax_level) {
  t <- dplyr::enquo(tax_level)
  tax_level.abund <- paste0(dplyr::quo_name(t), ".Abundance")

  melted_df %>%
    dplyr::group_by(SampleID, !!sym(tax_level)) %>%
    dplyr::filter(Abundance > 0) %>%
    dplyr::summarize(!!tax_level.abund := sum(as.numeric(Abundance)), TreatmentGroup = dplyr::first(TreatmentGroup)) %>%
    stats::na.omit() %>%
    dplyr::mutate(Relative.Abundance = 100 * !!sym(tax_level.abund) / sum(!!sym(tax_level.abund)))
}
```

Reply:
Awesome! I'll check it out and update it and the documentation for it.

What about `melt_proportions`?

Reply:
Or `convert_proportions`?

Reply:
See https://github.com/vallenderlab/MicrobiomeR/pull/59/commits/b4391d059889aff6ead1dfcb040af469deea692f